druzhba/postgres.py

```diff
@@ -155,12 +184,32 @@ def query_to_redshift_create_table(self, sql, table_name):
                     table_name,
                 )
 
-            desc = self.get_sql_description(sql)
+            table_attributes, columns = self.get_sql_description(sql)
             create_table = """CREATE TABLE "{}"."{}" (\n    """.format(
                 self.destination_schema_name, table_name
             )
-            field_strs = []
-            for (name, type_code, _, internal_size, precision, scale, null_ok,) in desc:
+            field_strs, comments = [], []
+            table_comment = table_attributes.get("comment")
+            if table_comment is not None:
+                comments.append(
+                    """COMMENT ON TABLE "{}"."{}" IS '{}'""".format(
```

Comment:
huh, weird that  redshift quotes `'` as ... `''`. 

you could use `$$ ... $$` (any number of $ allowed) to delimit the comment string, it isn't documented anywhere I can find but works in Redshift.